apollo-federation/src/sources/connect/validation/graphql/strings.rs

```diff
@@ -109,7 +109,7 @@ mod tests {
         };
         let field = query.fields.get("field").unwrap();
         let directive = field.directives.get("connect").unwrap();
-        directive.argument_by_name(name).unwrap()
+        directive.argument_by_name(name, schema).unwrap()
```

Comment:
```suggestion
        directive.specified_argument_by_name(name).unwrap()
```
there's actually a new api for this that doesn't require a schema

Reply:
i hit this on the merge pr: https://github.com/apollographql/router/pull/6058/commits/3ac1a72bb463d619e6587372f07a864d88eaa90b